riak_test/opbcounter_test.erl

```diff
@@ -33,7 +37,7 @@ confirm() ->
     decrement_test(Nodes),
     transfer_test(Nodes),
     conditional_write_test(Nodes),
-    rt:clean_cluster(Nodes),
+    %rt:clean_cluster(Nodes),
```

Comment:
It was just in some tests. I decided to remove it since it was not affecting the tests (from what I know) and it was slowing it down.

Manuel
> On 14 Sep 2015, at 13:45, Annette Bieniusa <notifications@github.com> wrote:
> 
> In riak_test/opbcounter_test.erl <https://github.com/SyncFree/antidote/pull/143#discussion_r39388724>:
> 
> > @@ -33,7 +37,7 @@ confirm() ->
> >      decrement_test(Nodes),
> >      transfer_test(Nodes),
> >      conditional_write_test(Nodes),
> > -    rt:clean_cluster(Nodes),
> > +    %rt:clean_cluster(Nodes),
> Why is the call to rt:clean_cluster no longer needed? (Here and in the other test cases..)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/SyncFree/antidote/pull/143/files#r39388724>.
>